src/app/(post-login)/submission/program/[shortName]/clinical-submission/page.tsx

```diff
@@ -451,4 +452,9 @@ const ClinicalSubmissionPage = ({ params: { shortName } }: { params: { shortName
 	}
 };
 
-export default ClinicalSubmissionPage;
+export default pageWithPermissions(ClinicalSubmissionPage as React.ComponentType, [
```

Comment:
usage of `as` due to a rabbit hole with EmotionJSX.Element typings
tried to use `React.ComponentType` directly but we are using the Emotion compiler for our tsx. This means all our components return `EmotionJSX.Element | undefined`. I can get the typings working until the `pageWithPermissions` higher order function, but it complains on rendering `<Page>`

Reply:
https://github.com/icgc-argo/rdpc-ui/pull/108/commits/9e68132df235548f829b056e7b1d482949fc07fb

```typescript
type Component = (...args: any[]) => EmotionJSX.Element; // this works as a component type
```
still some kinks to iron out. Typescript doesn't like implicit returns in functions. This can be toggled as a config but I'd rather try and make it work first. so if we have Components operating with implicit returns like this:
```typescript
type Component = (...args: any[]) => EmotionJSX.Element | undefined; 
```
Typescript won't like the render call when it comes to `<Component/>`
There's probably a way to call a component function to check for `undefined` return type but this seems incorrect to me. 

The way around it is no implicit returns but then you account for all cases and can end up with ugly filler: 
https://github.com/icgc-argo/rdpc-ui/pull/108/commits/9e68132df235548f829b056e7b1d482949fc07fb#diff-484f563422813ea468a6ce776b24d347c5dd72d9e792d0e2b1d1786992a7fdd6R453